src/widget/textbox.rs

```diff
@@ -312,17 +325,26 @@ impl Widget<String> for TextBox {
         _layout_ctx: &mut LayoutCtx,
         bc: &BoxConstraints,
         _data: &String,
-        _env: &Env,
+        env: &Env,
     ) -> Size {
-        bc.constrain((self.width, BOX_HEIGHT))
+        // QUESTION: is this the right place to put this default width?
+        let default_width = 100.0;
```

Comment:
The slider and the progress bar each have a default width for when they're given an infinite width to fill. Is it okay in here, or should it be part of the theme?

Reply:
I'm personally ok leaving this; it seems useful for prototyping but probably in a "real" layout it'll be constrained by parent. I think we can revisit though.